cocktails9/cocktails9/Model/ResponseList.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+struct CategoriesResponse: Codable {
+    let drinks: [Category]
+}
+
+struct Category: Codable {
+    let strCategory: String
```

Comment:
Not a big deal but it would be nicer to have these used without "str" part, like in Cocktail model